packages/cms/src/studio/schema-types/changelog-type.ts

```diff
@@ -0,0 +1,41 @@
+import { defineField, defineType } from 'sanity';
+
+export const changelogType = defineType({
```

Comment:
Possibly this should contain a `version` field?

Reply:
Not sure exactly how this would work. What if a feature is released on mobile and extension? Should it be an array of versions?